### PR TITLE
Updates after sync with PM

### PR DESCRIPTION
**Pull request by tweekerz**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/unlimited-archiving.md [hunk 1]
```diff
@@ -85,7 +85,8 @@ Here are some things to consider when using Outlook or Outlook on the web to acc
...
+- Search for auto-expanded archiving is available in Outlook for the web. Similar to Online Archive, you can search for items that were moved to an additional storage area only by searching the current folder itself. This means that you must select the archive folder in the folder list, and then select a single folder as your search scope. Similarly, if a folder in an auto-expanded storage area contains subfolders, you must search each subfolder separately.
+  - Auto-expanded archive search is available in Outlook Desktop in Current Channel (Preview). Within this preview, the Current Mailbox scope is available, thus allowing you to search the auto-expanded archive. For more information about this and other Microsoft Search support features, see https://techcommunity.microsoft.com/t5/outlook-global-customer-service/how-outlook-for-windows-connected-to-exchange-online-utilizes/ba-p/1715045.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/unlimited-archiving.md]:
```suggestion
- Auto-expanded archive search is available in Outlook Desktop in Current Channel (Preview). Within this preview, the Current Mailbox scope is available, thus allowing you to search the auto-expanded archive. For more information about this and other Microsoft Search support features, see [How Outlook for Windows connected to Exchange Online utilizes Microsoft Search](https://techcommunity.microsoft.com/t5/outlook-global-customer-service/how-outlook-for-windows-connected-to-exchange-online-utilizes/ba-p/1715045). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@tweekerz

**Comment by tweekerz** [comment]
@yogkumgit approved please commit the PR. Thanks!

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tweekerz** [comment]
Do we have any ETA on the PR? I still do not see it approved.

Rob Whaley<mailto:rob.whaley@microsoft.com>
Sr. Support Escalation Engineer, Beta Engineer

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by markjjo** [comment]
@tweekerz PR merged. Updated article should be live later today or tomorrow mid-AM at the latest.

**Comment by tweekerz** [comment]
Thanks Mark!
________________________________

**Pull request merged**